cloud/pkg/admissioncontroller/admit_rule.go

```diff
@@ -41,8 +41,8 @@ func admitRule(review admissionv1beta1.AdmissionReview) *admissionv1beta1.Admiss
 		reviewResponse.Allowed = true
 		return &reviewResponse
 	default:
-		err := fmt.Errorf("Unsupported webhook operation %v", review.Request.Operation)
-		klog.Errorf("Unsupported webhook operation %v", review.Request.Operation)
+		err := fmt.Errorf("unsupported webhook operation %v", review.Request.Operation)
+		klog.Errorf("unsupported webhook operation %v", review.Request.Operation)
```

Comment:
I think it's better to keep the first letter capitalized in klog.

Reply:
I agree with you.
But it looks like @fisherxu doesn't think so.

Reply:
ref: https://github.com/kubeedge/kubeedge/pull/4011/files#diff-364f053b5a428af19fb62f64c0c1675599b8d5ddc316c75bc71e25fc6340e54aR36